pkg/controller/certificaterequest/issue_certificate.go

```diff
@@ -34,10 +35,14 @@ import (
 	"github.com/openshift/certman-operator/pkg/localmetrics"
 )
 
+const (
+	leMaintMessage = "The service is down for maintenance or had an internal error."
```

Comment:
Is there something like an HTTP status code that could give us this status without relying on string matching? (this is obv way better than nothing, but wondering if there is something programmatic that is more robust)

Reply:
I'm honestly not sure. The output captured in [OSD-8663](https://issues.redhat.com/browse/OSD-8663) suggests maybe it's error code 0, but that doesn't seem right to me? I'm not sure how to go about finding out what error code might be generated by the server, either. Maybe search through any open source ACME server code?